include/core/lf_types.h

```diff
@@ -72,7 +72,12 @@ typedef enum {defer, drop, replace} lf_spacing_policy_t;
  * ignore freeing the void* inside a token if the void*
  * value is garbage collected by an external controller
  */
-typedef enum {no=0, token_and_value, token_only} ok_to_free_t;
+typedef enum {
```

Comment:
```suggestion
typedef enum {
    neither_token_nor_value=0,
	value_only,
    token_only,
	token_and_value
} ok_to_free_t;
```

Reply:
ok_to_free_t flag = token_and_value;

    if (flag == token_and_value) {
        /* Do the stuff when both value and token flag is set */
    }

    if (flag & value) {
        /* Do the stuff when value flag is set */
    }

    if (flag & token) {
        /* Do the stuff when token flag is set */
    }

    if (flag == neither_token_nor_value) {
        /* Do the stuff when neither is set */
    }